packages/shared/src/utils/validation/declarative-validation-rules.ts

```diff
@@ -23,7 +23,11 @@ export const validAddress = (value: string) => !/[`~!$%^&*_=+[}{\]\\"?><|]+/.tes
 export const validPostCode = (value: string) => value === '' || /^[A-Za-z0-9][A-Za-z0-9\s-]*$/.test(value);
 export const validTaxID = (value: string) => /(?!^$|\s+)[A-Za-z0-9./\s-]$/.test(value);
 export const validPhone = (value: string) => /^\+?([0-9-]+\s)*[0-9-]+$/.test(value);
-export const validLetterSymbol = (value: string) => /^[A-Za-z]+([a-zA-Z.' -])*[a-zA-Z.' -]+$/.test(value);
+export const validLetterSymbol = (value: string) => {
+    const pattern = "^[A-Za-z]+([a-zA-Z.' -])*[a-zA-Z.' -]+$";
+    const regex = new RegExp(pattern);
+    regex.test(value);
```

Comment:
i think you need to return value of this test.

Reply:
We are returning the validity of the string in this function.I will check the sonarcloud issue and add checklist for QA too